src/context/IpfsContext.tsx

```diff
@@ -0,0 +1,6 @@
+import { IPFSHTTPClient } from "ipfs-http-client";
+import React from "react";
+
+const IpfsContext = React.createContext<IPFSHTTPClient | null>(null);
```

Comment:
Why are we still having this file now? Are we using the global cached client or we are using a client context?

Reply:
Yes I created a context. But it is now used the way we want. See, when ever we create a campaign and stage we redirect to that page which reloads the application. So, it does reinitiate everything and we are only using it in two places.

Reply:
Need to get rid of this file and all references as it is no longer needed.